### PR TITLE
Fix builders tag images

### PR DESCRIPTION
**Pull request by jayunit100**
A couple more fixes for the builders, removing tag-images

### CODE REVIEW

**Comment by jayunit100** [comment]
@Brian-McM can we merge and itereate on this ? that way at least we can build from master as normal :) then maybe if need to add `tag-images` back can do that later ?

**Comment by jayunit100** [comment]
ok, just pushed the `retag-build-images-with-registries`  .... appears to work nicely for me :) 

```
calico/node                                   latest-amd64                            a42937029471   13 seconds ago   169MB
calico/node                                   v3.20.0-0.dev-26-g6cd4484-dirty-amd64   a42937029471   13 seconds ago   169MB
quay.io/calico/node                           v3.20.0-0.dev-26-g6cd4484-dirty         a42937029471   13 seconds ago   169MB
```

File: Makefile [hunk 1]
```diff
@@ -142,8 +142,12 @@ LOCAL_BUILD=true
 .PHONY: dev-image dev-test dev-clean
 ## Build a local version of Calico based on the checked out codebase.
 dev-image: $(addsuffix -dev-image, $(filter-out calico felix, $(RELEASE_REPOS)))
+
+# Dynamically declare new make targets for all calico subprojects...
 $(addsuffix -dev-image,$(RELEASE_REPOS)): %-dev-image: ../%
-	@cd $< && export TAG=$$($(TAG_COMMAND)); make image tag-images \
+	echo "TARGET:"
+	echo $< 
+	@cd $< && export TAG=$$($(TAG_COMMAND)); make image retag-build-images-with-registries ARCHES=amd64 \
```
Comment (Brian-McM) [inline Makefile:150]: I think `TAG` needs to be `IMAGETAG`

**Review by Brian-McM — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (jayunit100) [reply Makefile:150]: Ah ok
Will fix

Comment (jayunit100) [reply Makefile:150]: looks like BUILD_IMAGE and IMAGETAG are Both required?

Comment (jayunit100) [reply Makefile]: yeah , its working now.

Comment (jayunit100) [reply Makefile]: IMAGETAG =s set at the bottom of the make declaration ......   i think all set now  ? 

seems to pass for me

File: Makefile [hunk 2]
```diff
@@ -142,12 +142,17 @@ LOCAL_BUILD=true
...
-	@cd $< && export TAG=$$($(TAG_COMMAND)); make image tag-images \
+	echo "TARGET:"
+	echo $< 
+	@cd $< && export TAG=$$($(TAG_COMMAND)); make image retag-build-images-with-registries \
+		ARCHES=amd64 \
 		BUILD_IMAGE=$(REGISTRY)/$* \
 		PUSH_IMAGES=$(REGISTRY)/$* \
 		LOCAL_BUILD=$(LOCAL_BUILD) \
-		IMAGETAG=$$TAG
+		IMAGETAG=$$TAG \
```
Comment (caseydavenport) [inline Makefile:155]: @jayunit100 is this right? Surprised we need a continuation here since it's the last line in the command.

Comment (jayunit100) [reply Makefile]: ah, yeah, 
no-op but cleaned up anyway

**Review by caseydavenport — APPROVED** [review]

**Pull request merged**